### PR TITLE
bug: Picklists performance

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**
Load time for 5000 Picklists reduced from 7.56s to 0.03s (when paginated) or 0.27s (when not paginated for input lists)

### CODE REVIEW

File: templates/teams/picklists.html [hunk 1]
```diff
@@ -72,6 +72,7 @@ <h4 class="govuk-heading-s">Deactivated</h4>
 					{% include "includes/information-text.html" with text="There aren't any items in this picklist" %}
 				{% endif %}
 			</div>
+			{% include "includes/pagination.html" with data=data %}
```
Comment (jfaracik) [inline templates/teams/picklists.html:75]: data=data isn't necessary

Comment (jfaracik) [reply templates/teams/picklists.html:75]: Also have the end to end tests been updated to handle pagination?

Comment (BenAndrew-Kainos) [reply templates/teams/picklists.html:75]: Removed data=data & the tests rely on the latest picklist being at the top (which it still is) so no changes required

File: templates/teams/picklists.html [hunk 2]
```diff
@@ -72,6 +72,7 @@ <h4 class="govuk-heading-s">Deactivated</h4>
 					{% include "includes/information-text.html" with text="There aren't any items in this picklist" %}
 				{% endif %}
 			</div>
+			{% include "includes/pagination.html" %}
```
[the remark this answers is not in the document]
Comment (BenAndrew-Kainos) [reply templates/teams/picklists.html:75]: done :)

**Pull request merged**